libs/platform/src/lib/components/table/components/table-p13-dialog/sorting/sorting.component.html

```diff
@@ -0,0 +1,118 @@
+<fd-dialog class="fdp-table-p13-sorting-dialog">
+    <fd-dialog-header>
+        <ng-template fdTemplate="header">
+            <div fd-bar-left>
+                <fd-bar-element>
+                    <h1
+                        fd-title
+                        i18n="@@platformTableP13SortDialogHeader"
+                    >Sort</h1>
+                </fd-bar-element>
+            </div>
+            <div fd-bar-right>
+                <fd-bar-element>
+                    <fdp-table-reset-button></fdp-table-reset-button>
+                </fd-bar-element>
+            </div>
+        </ng-template>
+    </fd-dialog-header>
+
+    <fd-dialog-body>
+
+        <div
+            class="sort-row"
+            *ngFor="let rule of rules; let i = index"
+        >
+            <fd-select
+                class="sort-row__select"
+                [value]="rule.columnKey"
+                (valueChange)="_onRuleColumnKeyChange(rule, $event)"
+                [compact]="true"
+                placeholder="(none)"
+                i18n-placeholder="@@platformTableP13SortDialogNoneSelected"
+            >
+                <fd-option
+                    *ngFor="let column of columns"
+                    [value]="column.key"
+                >
+                    {{column.label}}
+                </fd-option>
+            </fd-select>
+
+            <fd-select
+                class="sort-row__select"
+                [value]="rule.direction"
+                (valueChange)="_onRuleDirectionChange(rule, $event)"
+                [compact]="true"
+                placeholder="(none)"
+                i18n-placeholder="@@platformTableP13SortDialogNoneSelected"
+            >
+                <fd-option
+                    [value]="SORT_DIRECTION.ASC"
+                    i18n="@@platformTableSortDialogSortOrderAsc"
+                >
+                    Ascending
+                </fd-option>
+                <fd-option
+                    [value]="SORT_DIRECTION.DESC"
+                    i18n="@@platformTableSortDialogSortOrderAsc"
+                >
+                    Descending
+                </fd-option>
+            </fd-select>
+
+            <div class="sort-row__actions">
+                <button
+                    fd-button
+                    fdType="transparent"
+                    glyph="sys-cancel"
+                    [compact]="true"
+                    aria-label="Remove"
+                    title="Remove"
+                    (click)="_removeRule(rule)"
+                ></button>
+
+                <button
+                    fd-button
+                    fdType="transparent"
+                    glyph="add"
+                    [compact]="true"
+                    aria-label="Add new"
+                    title="Add new"
```

Comment:
The same about `i18n`.
Also, you forgot about `i18n-aria-label`.

Reply:
added